lib/was_crawl_preassembly/utilities.rb

```diff
@@ -2,9 +2,8 @@ module Dor
   module WASCrawl
     class Utilities
       def self.get_collection_id(druid_obj)
-        collections = druid_obj.collections
-        raise "#{druid_obj.id} doesn't belong to a collection" if collections.length == 0
-        collection = collections[0]
+        collection = druid_obj.collections.first
```

Comment:
So are you saying the risk is very low?  I think the code as you've written it is clear;  I just want to avoid getting NoMethodError if  `.first` is called on nil, or if .collections is called on nil -- because we could give more context than a generic ruby error.

If `druid_obj.collections`  can never be nil, and if  `[].first` is not going to barf, then the code as is stands.  

Given that we are changing this because ActiveFedora changed under us ... I'm wondering where the sweet spot is here.